packages/atomic/src/components/common/image-carousel/image-carousel.tsx

```diff
@@ -0,0 +1,74 @@
+import {h, FunctionalComponent, Fragment} from '@stencil/core';
+import {JSXBase} from '@stencil/core/internal';
+import ArrowRight from '../../../images/arrow-right.svg';
+import {Button} from '../button';
+import {AnyBindings} from '../interface/bindings';
+import CarouselIndicator from './image-carousel-indicators';
+
+export interface CarouselProps {
+  bindings: AnyBindings;
+  previousImage(): void;
+  nextImage(): void;
+  navigateToImage(index: number): void;
+  numberOfImages: number;
+  currentImage: number;
+}
+
+export const ImageCarousel: FunctionalComponent<
+  CarouselProps & JSXBase.HTMLAttributes<HTMLHeadingElement>
+> = (props, children) => {
+  const commonPaginationClasses =
+    'w-6 h-6 grid justify-center items-center z-1 group rounded-full duration-200 opacity-50 hover:opacity-100 focus:opacity-100 transition-opacity hover:shadow bottom-0 mb-1 absolute';
```

Comment:
Ideally I would want the opacity to apply only to the background and not the arrows, but our tailwind theme is not compatible with opacity modifiers on custom colors.